tagsets/helpers.php

```diff
@@ -15,25 +15,26 @@ function support_mail_link() {
 }
 
 function multi_array_sort(&$data, $sortby) {
-    static $sort_funcs = array();
-
+    $sortby1=""; $sortby2=""; $sortby3="";
```

Comment:
I think you can just use $sorting_varN from the beginning (i.e. avoid redefining).

Reply:
Ok. Replaced $sortbyN by $sorting_varN at the beginning and deleted assignments later.